lark/lark.py

```diff
@@ -371,19 +376,17 @@ def _load(self, f, transformer=None, postlex=None):
             data['parser'],
             memo,
             self._callbacks,
-            self.options.postlex,
-            self.options.transformer,
             re_module,
-            self.options.debug
+            self.options, # Not all, but multiple attributes are used
```

Comment:
This comment raises more questions than it answers.  :-D

I'd move it from here entirely and put it up to right where you define `_LOAD_ALLOWED_OPTIONS`, with more explanation.  I tried to write that longer explanation but couldn't, which means it needs it.

Reply:
This has nothing to do with `_LOAD_ALLOWED_OPTIONS`, but with `parser_frontend.WithLexer.deserialize` which takes some options and assigns them to attributes. (Including as example `use_bytes`) Previously they were all passed as arguments, leading to like five extra arguments. But once again, maybe we should do it more explicit here.

Reply:
On third through, my judgement was probably wrong. Using it is less pythonic.

Reply:
@erezsh Went in the other direction, so we keep this.